src/printers/object.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ */
+
+'use strict';
+
+import Tokens from '../Tokens';
+
+import {printList} from '../utils';
+
+export default {
+  ObjectProperty: ({print, node}) => [
+    node.computed
+      ? [
```

Comment:
I can't wait to make more progress and run the formatter on its own source to see what it does :P